ui/src/components/ThemeSwitcher.tsx

```diff
@@ -2,7 +2,7 @@ import { createSignal, createEffect, JSXElement } from 'solid-js'
 
 export function ThemeSwitcher(): JSXElement {
   const [isDark, setIsDark] = createSignal<boolean>(
-    JSON.parse(localStorage.getItem('isDark')) ?? false
+    JSON.parse(localStorage.getItem('isDark') ?? 'false') ?? false
```

Comment:
Aahhh here it went wrong. I think we can remove the outer  `?? false` then right? That was meant to do this.